<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QTabBar::tab {width: 128px; height: 48px; font: 14px;}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="HomeTab" name="tab">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <widget class="QPushButton" name="exportToSCADButton">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>EXPORT TO SCAD</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="useExistingKeyButton">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>USE EXISTING KEY</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="visualizeSTLButton">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>VISUALIZE STL</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="importParametersButton">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>IMPORT PARAMETERS FROM TXT</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="exportParametersButton">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>EXPORT PARAMETERS TO TXT</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="createKeyButton">
            <property name="minimumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>256</width>
              <height>64</height>
             </size>
            </property>
            <property name="text">
             <string>CREATE KEY</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Menu 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="openAllButtonMenu1">
            <property name="text">
             <string>OPEN ALL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeAllButtonMenu1">
            <property name="text">
             <string>Close All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllButtonMenu1">
            <property name="text">
             <string>Clear All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="LayoutMgr" name="menu1LayoutMgr" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Menu 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="openAllButtonMenu2">
            <property name="text">
             <string>Open All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeAllButtonMenu2">
            <property name="text">
             <string>Close All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllButtonMenu2">
            <property name="text">
             <string>Clear All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="LayoutMgr" name="menu2LayoutMgr" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Menu 3</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="openAllButtonMenu3">
            <property name="text">
             <string>Open All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closeAllButtonMenu3">
            <property name="text">
             <string>Close All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearAllButtonMenu3">
            <property name="text">
             <string>Clear All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="LayoutMgr" name="menu3LayoutMgr" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_11">
        <item row="0" column="0">
         <widget class="LayoutMgr" name="menuSettingsLayoutMgr" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>STL Viewer</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="stlViewerLayout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LayoutMgr</class>
   <extends>QWidget</extends>
   <header>layoutmgr.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
